hub/api/tests/test_api.py

```diff
@@ -0,0 +1,64 @@
+import os, shutil
+import numpy as np
+from hub.api.dataset import Dataset
+from hub.core.storage import LocalProvider
+from hub.core.chunk_engine.read import read_dataset_meta, read_tensor_meta
+from hub.core.tests.common import parametrize_all_dataset_storages
+
+import pytest
+
+
+def test_create_empty_local_dataset():
+    ds = Dataset("/tmp/hub-test/empty")
```

Comment:
can you somehow derive this path from how the `ds`/`storage` fixtures work? i think you're not using them here to have code coverage of the constructors? that's fine but would be nice to be synchronized with the fixtures

Reply:
shoot, forgot to do that. my bad